targets/WindowsSdk/source/source/PlayFabSettings.cpp.ejs

```diff
@@ -2,14 +2,14 @@
 
 namespace PlayFab
 {
-    const utility::string_t PlayFabSettings::sdkVersion = U("<%- sdkVersion %>");
-    const utility::string_t PlayFabSettings::buildIdentifier = U("<%- buildIdentifier %>");
-    const utility::string_t PlayFabSettings::versionString = U("WindowsSDK-<%- sdkVersion %>");
+    const utility::string_t PlayFabSettings::sdkVersion = L"<%- sdkVersion %>";
+    const utility::string_t PlayFabSettings::buildIdentifier = L"<%- buildIdentifier %>";
+    const utility::string_t PlayFabSettings::versionString = L"WindowsSDK-<%- sdkVersion %>";
 
     bool PlayFabSettings::useDevelopmentEnvironment = false;
     utility::string_t PlayFabSettings::serverURL;
-    utility::string_t PlayFabSettings::developmentEnvironmentURL = U(".playfabsandbox.com");
-    utility::string_t PlayFabSettings::productionEnvironmentURL = U(".playfabapi.com");
+    utility::string_t PlayFabSettings::developmentEnvironmentURL = L".playfabsandbox.com";
```

Comment:
Should this be const?

Reply:
I don't think that even exists anymore. It's just an ancient hanger-on.
Truthfully we should just delete it, and make sure there's easy private cluster redirect support (For all sdks).